vite.config.ts

```diff
@@ -0,0 +1,8 @@
+// vite.config.ts
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  test: {
+    globals: true,
```

Comment:
better not use globals

Reply:
This one implies importing `expect` and so on for each test file. No big deal but I don't quite get why it is preferred.

Reply:
It's less typing, and easier to migrate from the JS projects, when they do not have that line already.

Reply:
I'd argue that migration from Jest to Vitest should not be done while migrating projects from JS to TS.
I think it's important that we use the tools as they are intended. Vitest is not intended to be used with globals. The option is here to help migrate (large) projects from Jest.
Less typing: not if your editor is configured correctly (these should be auto-imported on first use).